Makefile

```
@@ -206,30 +206,30 @@ build:
+	cd cli && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags "-extldflags '-static'" -o ../$(BUILD_DIR)/cli ./cmd/cli
 
 INSTALL_DIR ?= $(shell go env GOPATH)/bin
 
-carbide-cli:
-	go build -o $(INSTALL_DIR)/carbidecli ./cli/cmd/carbidecli
-	@echo "Installed carbidecli to $(INSTALL_DIR)/carbidecli"
+nico-cli:
+	go build -o $(INSTALL_DIR)/cli ./cli/cmd/cli
+	@echo "Installed cli to $(INSTALL_DIR)/cli"
 
 docker-build:
```

checkmake (0.3.2)

[warning] Target body for "docker-build" exceeds allowed length of 5 lines (9).

(maxbodylength)